### PR TITLE
tools: support changing instance log level

### LINKED ISSUE #4511: set log level via pd-ctl succeed, but don't work.
### What did you do?
/var/lib/pd/log # /pd-ctl log debug
Success!
/var/lib/pd/log # tail -f pd.log
[2021/12/28 05:59:39.271 +00:00] [ERROR] [client.go:171] ["region sync with leader meet error"] [error="[PD:grpc:ErrGRPCRecv]rpc error: code = Unavailable desc = server not started"]
[2021/12/28 05:59:40.272 +00:00] [INFO] [client.go:167] ["server starts to synchronize with leader"] [server=tc-pd-0] [leader=tc-pd-2] [request-index=0]
[2021/12/28 06:32:40.747 +00:00] [WARN] [grpclog.go:60] ["grpc: Server.processUnaryRPC failed to write status: connection error: desc = \"transport is closing\""]
[2021/12/28 06:34:41.288 +00:00] [WARN] [grpclog.go:60] ["grpc: Server.processUnaryRPC failed to write status: connection error: desc = \"transport is closing\""]
[2021/12/28 06:43:58.963 +00:00] [WARN] [grpclog.go:60] ["transport: http2Server.HandleStreams failed to read frame: read tcp 10.233.86.141:2379->10.233.116.155:59226: read: connection reset by peer"]
[2021/12/28 06:48:27.824 +00:00] [WARN] [grpclog.go:60] ["grpc: Server.processUnaryRPC failed to write status: connection error: desc = \"transport is closing\""]
[2021/12/28 06:59:00.694 +00:00] [INFO] [index.go:189] ["compact tree index"] [revision=1]
[2021/12/28 06:59:00.695 +00:00] [INFO] [kvstore_compaction.go:55] ["finished scheduled compaction"] [compact-revision=1] [took=34.19µs]
[2021/12/28 07:59:00.707 +00:00] [INFO] [index.go:189] ["compact tree index"] [revision=2414]
[2021/12/28 07:59:00.734 +00:00] [INFO] [kvstore_compaction.go:55] ["finished scheduled compaction"] [compact-revision=2414] [took=26.781576ms]

### What version of PD are you using (`pd-server -V`)?
/ # /pd-server -V
Release Version: v5.4.0-nightly
Edition: Community
Git Commit Hash: ea57c6d6a0c9ad081aded76545f3d97bc465635f
Git Branch: heads/refs/tags/v5.4.0-nightly
UTC Build Time:  2021-12-28 04:09:10

### PR DESCRIPTION
**Pull request by rleungx**
### What problem does this PR solve?

Close #4511. This PR can be reviewed after #4571 is merged.

### What is changed and how it works?

This default behavior isn't changed and after this PR, we can use the following method to change the instance log level:
```
>> log debug 172.16.4.3:37810
Success!
```

### Check List

Tests

- Unit test

### Release note

```release-note
None
```

### CODE REVIEW

File: tools/pd-ctl/pdctl/command/log_command.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ var (
 // NewLogCommand New a log subcommand of the rootCmd
 func NewLogCommand() *cobra.Command {
 	conf := &cobra.Command{
-		Use:   "log [fatal|error|warn|info|debug]",
+		Use:   "log [fatal|error|warn|info|debug] [addr]",
```
Comment (bufferflies) [inline tools/pd-ctl/pdctl/command/log_command.go:32]:
```suggestion
		Use:   "log [fatal|error|warn|info|debug] [addrs]",
```

**Comment by bufferflies** [comment]
It will be better if you can add one function that shows all endpoint log level, it will be helpful to write unit test.

File: tools/pd-ctl/pdctl/command/log_command.go (hunk 1 above)
Comment (bufferflies) [inline tools/pd-ctl/pdctl/command/log_command.go:32]: May I set all endpoints's log  to be info.

Comment (rleungx) [reply tools/pd-ctl/pdctl/command/log_command.go:32]: No, It's not easy to get all PD members with a single request.

File: tests/pdctl/log/log_test.go [hunk 2]
```diff
@@ -33,64 +33,110 @@ func Test(t *testing.T) {
 
 var _ = Suite(&logTestSuite{})
```
Comment (JmPotato) [inline tests/pdctl/log/log_test.go:34]: If the two test cases run concurrently, is there a possibility that this test is unstable?

**Review by JmPotato — COMMENTED** [review]
Overall LGTM. BTW, do we have some manual test results?

File: tests/pdctl/log/log_test.go (hunk 2 above)
Comment (rleungx) [reply tests/pdctl/log/log_test.go:34]: These two cases won't run concurrently.

**Review by JmPotato — APPROVED** [review]

**Review by nolouch — APPROVED** [review]
lgtm

**Pull request merged**